conda-forge.yml

```diff
@@ -2,4 +2,4 @@ channel_priority: strict
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
\ No newline at end of file
+      vmImage: windows-2019
```

Comment:
Please don't change this file as it's part of the overall infrastructure. It's just a new line so no big impact but it makes reviewing more difficult since the reviewer has to check if there is any substantial change or just something cosmetic. 
I'd just reverse the commit that caused that or copy/paste the file from master.

Reply:
Yes, the file is already there but your editor (I assume) added the newline and therefore changed it. I'd wait for the review team to have a look at this.

Reply:
I see. Ok.

Reply:
Removed modified `conda-forge.yml` from recipe-folder.

Reply:
I just copy-pasted the content of `conda-forge.yml` from the *main* branch.

Reply:
Looks good!